src/helpers/firebase/index.js

```diff
@@ -1,44 +1,43 @@
-import firebase from 'firebase'
+import firebase from "firebase";
 
 if (!!!firebase.apps || !firebase.apps.length) {
     firebase.initializeApp({
         apiKey: process.env.NEXT_PUBLIC_FIREBASE_KEY, // Auth / General Use
-        authDomain: '',
-        projectId: '',
-        storageBucket: '',
-        messagingSenderId: '',
-        appId: '',
-        measurementId: '',
-    })
+        authDomain: "lamesa-customer.firebaseapp.com",
+        projectId: "lamesa-customer",
+        storageBucket: "lamesa-customer.appspot.com",
+        messagingSenderId: "433312528962",
+        appId: "1:433312528962:web:af7c207ef63cef24f94bdb",
+        measurementId: "G-SQV7GFTKWD",
```

Comment:
ni usamos este arcchivo porque ni usamos el login con google, fb, etc., así que probablemente lo borre, pero por mientras dejo unas suggestions

authDomain y projectId no son muy secrets que digamos, pero en caso de que cambiemos de proyecto de firebase o algo por el estilo sería más fácil cambiar las configs y no tener que andar tocando código
```suggestion
        apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
        authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
        projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
```

con lo que usamos de firebase no necesitamos storageBucket, messagingSenderId, appId, ni measurementId, así que de paso lo volamos